Mage/src/main/java/mage/players/Player.java

```diff
@@ -834,7 +839,13 @@ default List<Integer> getMultiAmount(Outcome outcome, List<String> messages, int
 
     boolean addCounters(Counter counter, UUID playerAddingCounters, Ability source, Game game);
 
-    void removeCounters(String name, int amount, Ability source, Game game);
+    void removeCounters(String counterName, int amount, Ability source, Game game);
+
+    int getCountersCount(CounterType counterType);
+
+    int getCountersCount(String counterName);
+
+    int getCountersTotalCount();
```

Comment:
No needs in such new methods in `Player` class. Add additional methods in `Counters` class instead.